src/common/constants/dmca.json

```diff
@@ -0,0 +1,4 @@
+{
+  "authors": [],
+  "post_ids": [13359110, 29061561]
```

Comment:
it would be easier to have the posts with `author/permlink`, is authors id based too?

Reply:
no it isnt, but yeah i can change it to author/permlink

Reply:
`authors` are usernames.